### PR TITLE
[kvm/docs] Improve KVM setup instructions

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

### Description of PR

Summary: Improve KVM setup instructions
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
I received some feedback while helping people onboard to the KVM testbed:
1. Not all VMs have `ethtool` installed by default, which can cause some strange errors.
2. There was some confusion over how to access `sonic-mgmt` inside the docker container.
3. Certain VM configurations (namely Hyper-V) require nested virtualization to be enabled explicitly in order to run KVM VMs inside them.
4. It's not clear how to check if the tests/testbed is working after all of these setup steps are complete.

#### How did you do it?
1. Added `ethtool` to the setup script.
2. Added clarification for the mount point for `sonic-mgmt`.
3. Added an explicit mention of nested virt.
4. Added a step to run the `bgp_fact` test.

#### How did you verify/test it?
Verified that the `setup` scripts work as described, and are idempotent.

### CODE REVIEW

**Review by yxieca — APPROVED** [review]

**Comment by lguohan** [comment]
retest this please

**Review by lguohan — APPROVED** [review]

File: ansible/doc/README.testbed.VsSetup.md [hunk 1]
```diff
@@ -1,77 +1,100 @@
...
 $ gzip -d sonic-vs.img.gz
 $ mkdir -p ~/sonic-vm/images
 $ mv sonic-vs.img ~/sonic-vm/images
 ```
 
 ## Setup sonic-mgmt docker
+All testbed configuration steps and tests are run from a `sonic-mgmt` docker container. This container has all the necessary packages and tools for SONiC testing so that test behavior is consistent between different developers and lab setups.
 
-Run the `setup-container.sh` in the root directory of the sonic-mgmt repository:
+1. Run the `setup-container.sh` in the root directory of the sonic-mgmt repository:
```
Comment (lguohan) [inline ansible/doc/README.testbed.VsSetup.md:68]: there are some pre-requiste to run setup-container.sh, basically j2cli and docker-engine. can you add those dependency in the doc as well?"

Comment (daall) [reply ansible/doc/README.testbed.VsSetup.md:70]: Good call, added!

File: ansible/setup-management-network.sh [hunk 2]
```diff
@@ -1,35 +1,53 @@
...
-    brctl delbr br1
+echo "STEP 4: Checking for ethtool package..."
+if ! command -v ethtool; then
+    echo "ethtool not found, install ethtool"
+    apt-get install -y ethtool
+fi
+echo
+
+echo "STEP 5: Checking if bridge br1 already exists..."
+if ! brctl show br1; then
```
Comment (prsunny) [inline ansible/setup-management-network.sh:40]: If it returns the output as below, then this step fails. May be we should just add br1 even if it exists.. 

```
STEP 5: Checking if bridge br1 already exists...
bridge name	bridge id		STP enabled	interfaces
br1		can't get info No such device

```

**Pull request merged**